packages/app/src/components/search/SearchPage.tsx

```diff
@@ -58,7 +58,7 @@ const SearchPage = () => {
               <SearchFilter.Select
                 className={classes.filter}
                 name="kind"
-                values={['Component', 'Template']}
+                values={['Component', 'Template', 'Documentation']}
```

Comment:
I think techdocs misses a `DocumentType` filter here. We created ne internally, but had no time to contribute it yet. I think `kind` means entity kind here.